sdks/python/apache_beam/io/avroio.py

```diff
@@ -338,12 +351,15 @@ def split_points_unclaimed(stop_position):
           yield record
 
 
+_create_avro_source = _FastAvroSource
```

Comment:
I guess this is for not breaking imports in other files. Looks like it is in these two files: bigquery.py and bigquery_read_internal.py. So we may able to get rid of it. Not blocking though!

Reply:
Yep, exactly.